packages/web/src/pages/track-page/components/TrackRemixes.tsx

```diff
@@ -99,16 +100,16 @@ export const TrackRemixes = (props: TrackRemixesProrps) => {
     <Flex
       direction='column'
       gap='l'
-      w={isMobile || isCommentingEnabled ? '100%' : 720}
+      w={!isDesktop || isCommentingEnabled ? '100%' : 720}
     >
       <Flex
         row
         alignItems='center'
         gap='s'
-        justifyContent={!isMobile && !isCommentingEnabled ? 'center' : 'left'}
+        justifyContent={!isCommentingEnabled ? 'center' : 'left'}
```

Comment:
do we not need to check for isDesktop here anymore?

Reply:
yeah im realizing this incorrect, we want it left aligned always when comments are enabled